src/databricks/labs/ucx/source_code/linters/spark_connect.py

```diff
@@ -203,6 +195,7 @@ def __init__(self, is_serverless: bool = False):
         ]
 
     def lint(self, code: str) -> Iterator[Advice]:
-        tree = ast.parse(code)
-        for matcher in self._matchers:
-            yield from matcher.lint_tree(tree)
```

Comment:
removing this is a regression. see https://github.com/databrickslabs/ucx/pull/1824

Reply:
fixed